extension/sqlsmith/fuzzyduck.cpp

```diff
@@ -62,18 +62,25 @@ void FuzzyDuck::FuzzAllFunctions() {
 	EndFuzzing();
 }
 
-string FuzzyDuck::GenerateQuery() {
-	LogTask("Generating query with seed " + to_string(seed));
+idx_t FuzzyDuck::GenerateRandomNuber() {
 	auto &engine = RandomEngine::Get(context);
-	// set the seed
 	engine.SetSeed(seed);
```

Comment:
I've been thinking about this more, and I'm realizing the lines 
```
auto &engine = RandomEngine::Get(context); 
engine.SetSeed(seed);
```
should be in the FuzzyDuck constructor. This code sets the seed for the random engine, and we don't need to be continuously resetting it I think. Especially not every time we get a random number
```
auto &engine = RandomEngine::Get(context);
```
can also stay in the `GenerateRandomNumber()` function though, since you need the engine to get the next random number

Reply:
Did it in last commit ([646f021](https://github.com/hmeriann/duckdb/pull/3/commits/646f021d5492e4fa0b0ef4763f3505aa6943cbae))